add multi-tenancy public tutorial (#2393)
giantswarm/docs: src/content/tutorials/observability/data-exploration/accessing-grafana/_index.md

After selecting the right identity provider, you may run through an authentication workflow. However, if you are already authenticated within the current browser, this will be skipped and you should see the Home dashboard as a result.

> When you log in, you will be logged into the `Shared Org`, which contains a curated set of provided dashboards that are managed and accessible to everyone. If you want to learn more about organizations and multi-tenancy, we can only advise you to read the related [documentation]{{< relref "/tutorials/observability/multi-tenancy" >}}).

Reviewer: ```suggestion
At the initial log in the preselected Grafana organization is the so called `Shared Org`. This shared organization contains a curated set of managed dashboards that are accessible to everyone with access to Grafana. If you want to learn more about organizations and multi-tenancy, we can only advise you to read the related [documentation]{{< relref "/tutorials/observability/multi-tenancy" >}}).
```

> When you log in, you will be logged into the Shared Org. which contains a curated set of provided dashboards that we manage and that is accessible to everyone. If you want to learn more about organizations and multi-tenancy, we can only advise you to read the related [documentation]{{< relref "/tutorials/observability/multi-tenancy" >}}).

Reviewer 2: ```suggestion
When you log in, you will be logged into the `Shared Org`, which contains a curated set of provided dashboards that are managed and accessible to everyone. If you want to learn more about organizations and multi-tenancy, we can only advise you to read the related [documentation]{{< relref "/tutorials/observability/multi-tenancy" >}}).
```
